code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -79,13 +79,17 @@
 	xeno_caste = X
 
 	plasma_stored = xeno_caste.plasma_max
-	maxHealth = xeno_caste.max_health
+	maxHealth = xeno_caste.max_health * GLOB.xeno_stat_multiplicator_buff
 	health = maxHealth
 	setXenoCasteSpeed(xeno_caste.speed)
 	soft_armor = getArmor(arglist(xeno_caste.soft_armor))
 	hard_armor = getArmor(arglist(xeno_caste.hard_armor))
 	warding_aura = 0 //Resets aura for reapplying armor
 
+/mob/living/carbon/xenomorph/proc/set_new_buffed_stat()
```

Comment:
Autodoc
Also this "set" is wierd as this isnt actually a setter you can pass args to 
Consider changing it to pass an arg which will default to the multiplier buff glob or renaming it